### PR TITLE
[Cassandra] Change: test connection query

### PR DESCRIPTION
**Pull request by yershalom**
…onnection

### CODE REVIEW

File: redash/query_runner/cass.py [hunk 1]
```diff
@@ -85,6 +86,9 @@ def run_query(self, query, user):
             json_data = json.dumps(data, cls=JSONEncoder)
 
             error = None
+        except ReadTimeout as e:
+            error = e
```
Comment (arikfr) [inline redash/query_runner/cass.py]: Error needs to be a string. But why handle ReadTimeout in a special way?

File: redash/query_runner/cass.py [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
     from cassandra.auth import PlainTextAuthProvider
+    from cassandra import ReadTimeout
     enabled = True
 except ImportError:
     enabled = False
 
 
 class Cassandra(BaseQueryRunner):
-    noop_query = "SELECT * FROM system"
+    noop_query = "SELECT * FROM system.compactions_in_progress"
```
Comment (arikfr) [inline redash/query_runner/cass.py]: Every user has access to this? Is there something like `SELECT 1`?

Comment (yershalom) [inline redash/query_runner/cass.py]: No  `SELECT 1` for CQL.
Most of test queries goes in the same manner like `SELECT now() FROM system.local;`

File: redash/query_runner/cass.py (hunk 1 above)
Comment (arikfr) [inline redash/query_runner/cass.py]: What do you mean by pass it?

**Pull request merged**